src/Aggregator.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Algolia\SearchBundle;
+
+use Algolia\SearchBundle\Exception\EntityNotFoundInObjectID;
+use Algolia\SearchBundle\Exception\InvalidEntityForAggregator;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Serializer\Normalizer\NormalizableInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * @ORM\MappedSuperclass
```

Comment:
Why does this class have to be mapped in ORM? It's not being saved in the database separately, so forcing every extending class to use that same identifier seems wrong.

Reply:
You are right, thanks!